frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -756,6 +756,8 @@ function ReaderDictionary:startSdcv(word, dict_names, fuzzy_search)
                 table.insert(args, opt)
             end
         end
+        table.insert(args, "--")
```

Comment:
Please add a comment about that (before, or on the line itself):
`-- prevent word starting with a "-" to be interpreted as a sdcv option`

Reply:
done